### ISSUE TITLE
Return type void

### ISSUE THREAD

**Issue by samuelecarpene** [yajra/laravel-datatables#1367]
I updated to v8 of datatables but i have an erorr using orderColumn.

I'm using datatables to show list of users with role.
When click on column header for ordering some data i receive this error

`Type error: Return value of Yajra\DataTables\QueryDataTable::applyOrderColumn() must be an instance of Yajra\DataTables\void, none returned`

### DataTable class
 ```
class UsersDataTable extends DataTable
{

public function dataTable($query)
    {
        $dataTable = new EloquentDataTable($query);

        return $dataTable
          ->rawColumns(['action', 'conc', 'active'])
          ->orderColumn('role', 'role_user.role_id $1')
        ;
}


public function query(User $model)
    {
        $query = $model->newQuery()
            ->join('role_user', 'users.id', '=', 'role_user.user_id')->select(['users.*', 'role_user.role_id'])
            ->where('users.id', '!=', Auth::user()->id)
            ->withTrashed();
        return $this->applyScopes($query);
    }
}
```

I think the problem is the return void in the function definition (https://github.com/yajra/laravel-datatables/blob/8.0/src/QueryDataTable.php#L596) because return type void is implemented in php 7.1 as described here (https://wiki.php.net/rfc/void_return_type#php_rfcvoid_return_type).
If i remove the return type all work correctly.

I checked all code and it is the only one present so i think you can remove to keep compatibility with php 7.0.

Thank you.

- Operating System Ubuntu 16.04
- PHP Version 7.0.22
- Laravel Version 5.4.36
- Laravel-Datatables Version 8.0.1

**Comment by yajra** [comment]
@samuelecarpene thanks for reporting. Must be added by phpstorm while refactoring the code. Will release  a patch for this. Thanks!

**Comment by yajra** [comment]
Patch released on v8.0.2. Thanks!

**Comment by yajra** [comment]
BTW, you can also remove `return $this->applyScopes($query);` and just return the `$query` directly. Apply scopes are automatically added on the process.
